### DISCUSSION TITLE
Loss global cell vector direction info when convert between `fmt`

### DISCUSSION THREAD

**Discussion by ZLI-afk** [deepmodeling/dpdata#641] [Q&A]
When I try to equally convert atomic structure among different files using `dpdata`, the resulted output file always omit global cell vector direction information with a different re-oriented structure. Could you please help to check my script:
```python
import dpdata

sys = dpdata.System('POSCAR.in', fmt='vasp/poscar')
sys.to('vasp/poscar', 'POSCAR.out')
```

`dpdata` version:  v0.2.17

`POSCAR.in`:
```
Li1 In1 Ge1
1.0
   3.1638271070853445   -0.0000000000245034   -3.1959480025730982
   0.0000000001949454    3.1959480021439473   -3.1959480025730986
   3.1638271069389132    3.1959480021684508    0.0000000000000011
Li Ge In
1 1 1
direct
   0.0000000000000000    0.0000000000000000    0.0000000000000000 Li
   0.7500004204687248    0.7499999587811100    0.7499994148949382 Ge
   0.4999995428090565    0.4999999725207402    0.4999996099299587 In
```

`POSCAR.out`:
```
Li1 Ge1 In1 
1.0
4.4970974637736054e+00 1.2223083718601634e-16 -3.9399063393568465e-17 
2.2712613453386430e+00 3.9076257713257756e+00 1.4916278915459289e-16 
2.2258361184342657e+00 1.3201438413316640e+00 3.6778743055239835e+00 
Li Ge In 
1 1 1 
Cartesian
   0.0000000000    0.0000000000    0.0000000000
   6.7456466906    3.9208262760    2.7584035772
   4.4970944771    2.6138841840    1.8389357181
```

**Comment by ZLI-afk** [comment]
I have locate the spot at [15th line in vasp.py](https://github.com/deepmodeling/dpdata/blob/master/dpdata/plugins/vasp.py), where the registered post function `rot_lower_triangular`  seems do the re-rotation that I do not expect. Maybe It's better to add a controller to switch it off?

**Reply by wanghan-iapcm** [reply]
I agree, please contribute. Thank you.

**Reply by ZLI-afk** [reply]
Thanks, I have submitted a PR #644. Hope it can address this.
